### DISCUSSION TITLE
VB.NET API problem.

### DISCUSSION THREAD

**Discussion by JohnC1102** [Sellbrite/Sellbrite-API#32] [Q&A]
I have placed this in the issues section but this is my first time so I am placing the same message here.

I have a vb.net application running at a customer site in Ohio that has functioned without error since 2020..

this is the api GET url: [https://api.sellbrite.com/v1/orders\](https://api.sellbrite.com/v1/orders%5C)
We have vaild account token / Secret Key.

Beginning 2/19/2024 we started getting the following error:

System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel.
at System.Net.HttpWebRequest.GetResponse()

From my developer location, in Missouri the codes connects fine, as expected.

What surprises me is that if I open the browser at my customer site in Ohio and put in the url and put the account token and secret key in as user/password it connects fine and returns the orders. No SSL/TSL error message.

Does anyone have any insight in this area?

Thanks,

John

**Comment by rengler-godaddy** [comment]
Hey John, 

I know we discussed this on this thread https://github.com/Sellbrite/Sellbrite-API/issues/31 but I wanted to make sure you got a response here as well. If you see any further issues let us know.

**Comment by JohnC1102** [comment]
Thanks Jordan.

Yes I saw the thread #31 responses.

JC

John Collett
Collett Software Solutions
12847 Daylight Drive
Apartment # 1127
St. Louis, MO 63131
314-974-0873
________________________________
